spec/maestro_spec.rb

```diff
@@ -0,0 +1,51 @@
+ENV["RACK_ENV"] = "test"
+
+require "rspec"
+require "rack/test"
+require "json"
+require "pry-byebug"
+require_relative "../app"
+
+describe "Maestro" do
+  include Rack::Test::Methods
+
+  subject { post "/maestro", params }
+  let(:params) { { text: text } }
+  let(:text) { "" }
+
+  before { subject }
+
+  def app
+    Sinatra::Application
+  end
+
+  def json_response
+    JSON.parse(last_response.body)
+  end
+
+  describe "/maestro help" do
+    let(:text) { "help" }
+
+    it "returns the usage text" do
+      expect(last_response).to be_ok
```

Comment:
Where is `last_response` coming from?

Reply:
It's a rack-test thing: https://github.com/rack-test/rack-test. It's basically the same as `response` in a normal request spec

Reply:
yep. It's a dependency of rspec-rails or something